gateway-export-plugin/src/main/java/com/ca/apim/gateway/cagatewayexport/tasks/explode/filter/EntityFilterRegistry.java

```diff
@@ -8,22 +8,44 @@
 
 import javax.inject.Inject;
 import javax.inject.Singleton;
-import java.util.Collection;
-import java.util.Set;
-import java.util.TreeSet;
+import java.util.*;
 
-import static java.util.Collections.unmodifiableSet;
+import static java.util.Collections.unmodifiableCollection;
 
 @Singleton
 public class EntityFilterRegistry {
 
     private final Collection<EntityFilter> entityFilters;
+    private Map<String, EntityFilter> sortedFilters = new LinkedHashMap<>();
```

Comment:
This is temporary so we can avoid it as a member. Pass it as an argument to the method.

Reply:
done